astronomer/providers/great_expectations/operators/great_expectations.py

```diff
@@ -233,13 +233,7 @@ def default_batch_request_dict(self) -> Dict[str, Any]:
             "data_connector_name": data_connector_name,
             "data_asset_name": self.data_asset_name,
         }
-        # BigQuery needs a temp table to run on; it is assumed the table will
-        # be named the same as the data asset but with an added _temp suffix
-        if self.conn_type == "gcpbigquery":
-            self.batch_request_extra["batch_spec_passthrough"] = {
-                "bigquery_temp_table": f"{self.data_asset_name}_temp"
-            }
-        batch_request.update(self.batch_request_extra)
```

Comment:
Is this no longer needed for BigQuery? I can't tell if it's being handled somewhere else.